UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.cs

```diff
@@ -803,14 +803,11 @@ public void NotifyPlayer(NetworkConnection playerGameObject)
 	[Server]
 	private void UpdateOccupant()
 	{
-		if (occupiableDirectionalSprite != null && occupiableDirectionalSprite.HasOccupant)
+		if (buckleInteract != null && buckleInteract.OccupantPlayerScript != null)
```

Comment:
```suggestion
		if (hasBuckleInteract && buckleInteract != null && buckleInteract.OccupantPlayerScript != null)
```
New bool `hasBuckleInteract`, set to true in awake if the component was found.

Just a small performance fix as it prevents two null checks being constantly run